mnc/control.py

```diff
@@ -525,3 +542,11 @@ def stop_dr(self, recorders=None, t0='now', queue_number=0):
                 logger.info(f"recording on {recorder} stopped")
             else:
                 logger.warn(f"stopping recording on {recorder} failed: {response['response']}")
+
+def _core_weight_func(r: float) -> float:
+    return 1.0 if r < CORE_RADIUS_M else 0.0
```

Comment:
Do all antennas actually have a weighting of either 1 or 0, or am I misunderstanding how the output is being used?

Reply:
Yes they do have a weighting of either zero or one. See https://github.com/ovro-lwa/mnc_python/blob/main/mnc/xengine_beamformer_control.py#L312. It probably makes a bit more sense to actually use the flags argument I just added for selecting for core antennas....but that's more code than I'm willing to write....